tox.ini

```diff
@@ -10,7 +10,12 @@ commands =
     coverage combine
     coverage report
     coverage xml
-passenv = DATABASE_URL PORT aws_access_key_id aws_secret_access_key
+passenv = 
+    DATABASE_URL PORT 
+    aws_access_key_id 
+    aws_secret_access_key 
+    is_aws_mturk_requestor 
+    mturk_worker_id
```

Comment:
`mturk_worker_id` is stored in `tests/config.py` outside of version control, and is documented in the "running the tests" docs.